configure.ac

```diff
@@ -213,6 +213,27 @@ AC_LINK_IFELSE(
 
 AM_CONDITIONAL([WITH_PCAP_FIX], [test "$pcap_fix" = "yes"])
 
+# Check for dlopen and dlfcn.h
+AC_ARG_ENABLE(modules,
+    AS_HELP_STRING([--enable-modules],
+                   [Allow loading third-party modules at runtime]))
+AS_IF([test "x$enable_modules" == "xyes"], [
+  MY_CPPFLAGS="$MY_CPPFLAGS -DBM_ENABLE_MODULES"
```

Comment:
I believe "dynamic module loading" should be enabled by default if the dlopen dependencies are met. I don't see any drawback in doing this and for most users it may mean one less configure knob that they have to worry about it. The code which is guarded by the preprocessor flags would also be less likely to break if it's built by default. Thoughts?

Reply:
I definitely prefer it on by default as well. I was afraid you'd oppose the dlopen requirement,  but setting the default to be enabled if dlopen is already available sounds like the right way to go. Will change.